scripts/ci/stats/merged_prs.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# Copyright (c) 2024 Intel Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+import os
+from github import Github
+import argparse
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import bulk
+
+
+date_format = '%Y-%m-%d %H:%M:%S'
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawDescriptionHelpFormatter, allow_abbrev=False)
+
+    parser.add_argument('--pull-request', required=True, help='pull request number', type=int)
+    parser.add_argument('--repo', required=True, help='github repo')
+
+    return parser.parse_args()
+
+def gendata(data, index):
+    for t in data:
+        yield {
+                "_index": index,
+                "_source": t
+                }
+
+def main():
+    args = parse_args()
+    token = os.environ.get('GITHUB_TOKEN')
+    if not token:
+        sys.exit('Github token not set in environment, please set the '
+                 'GITHUB_TOKEN environment variable and retry.')
+
+    gh = Github(token)
+    json_list = []
+    if args.repo:
```

Comment:
drop? arg.repo is required and nothing works without gh_repo set anyway

Reply:
fixed